test/setup-jsdom.js

```diff
@@ -6,6 +6,7 @@ const JSDOM = new jsdom.JSDOM("<!doctype html><html><body></body></html>");
 global.window = JSDOM.window;
 global.document = window.document;
 global.navigator = global.window.navigator;
+JSDOM.reconfigure({ url: "http://server.test/" });
```

Comment:
Could you add a comment why you had to add this?

Reply:
sure ;) It's to do with the `KintoAdmin` component calling `syncHistoryWithStore` which needs a real url (and not the base `about:blank` from jsdom)